test/integration/indexer/links_lookup_test.rb

```diff
@@ -120,6 +125,9 @@ def test_extracts_parts_of_taxonomy
       "content_id" => grandparent_1_content_id,
       "base_path" => "/grandparent-1",
       "title" => "Grandparent 1",
+      "details" => {
```

Comment:
We needed to add the internal name to the tests because the method we used from the govuk_taxonomy_helpers gem looks for the internal_name of the content item: https://github.com/alphagov/govuk_taxonomy_helpers/blob/master/lib/govuk_taxonomy_helpers/publishing_api_response.rb#L26-L31